test/test.start-kafka-advertised-host.kafka.sh

```diff
@@ -6,15 +6,13 @@ testAdvertisedHost() {
 	# Given a hostname is provided
 	export KAFKA_ADVERTISED_HOST_NAME=monkey
 	export KAFKA_ADVERTISED_PORT=8888
-	export KAFKA_HOST_NAME=internal-monkey
-	export KAFKA_PORT=2222
 
 	# When the script is invoked
 	source "$START_KAFKA"
 
 	# Then the configuration file is correct
-	assertExpectedConfig 'advertised.listeners=PLAINTEXT://monkey:8888'
-	assertExpectedConfig 'listeners=PLAINTEXT://internal-monkey:2222'
+	assertAbsent 'advertised.listeners'
+	assertExpectedConfig 'listeners=PLAINTEXT://monkey:8888'
```

Comment:
This needs fixing. If the user provides Advertised Host then this should set the advertised.listeners. It will break existing configs using ADVERTISED_HOST_NAME as the brokers wont' be able to resolve 'monkey' in the above case.